authentik/blueprints/apps.py

```diff
@@ -61,7 +63,19 @@ def _reconcile(self, prefix: str) -> None:
             except (DatabaseError, ProgrammingError, InternalError) as exc:
                 self.logger.warning("Failed to run reconcile", name=name, exc=exc)
 
-    def reconcile_tenant(self) -> None:
+    @staticmethod
+    def reconcile_tenant(func: Callable):
+        """Mark a function to be called on startup (for each tenant)"""
+        setattr(func, "_authentik_managed_reconcile", ManagedAppConfig.RECONCILE_TENANT_CATEGORY)
```

Comment:
If there are many reconcile functions for this category, would one get overridden?

Reply:
Yeah you can only have one category per reconcile function but that also matches the current behaviour

Reply:
Fair enough, I didn't look into the code too much

Reply:
And if needed we can always turn it into a list in the future